### PR TITLE
revised law jurisdictions

### PR DESCRIPTION
**Pull request by tahmeedshah**

### CODE REVIEW

File: app/views/forms/view/bch/view-biosafety-law.directive.html [hunk 1]
```diff
@@ -26,29 +26,27 @@ <h4>
...
 				<div class="km-value">
 					<span ng-if="item.identifier!='5B6177DD-5E5E-434E-8CB7-D63D67D5EBED'" ng-bind="document.type|term:locale"></span>
 					<span ng-if="item.identifier=='5B6177DD-5E5E-434E-8CB7-D63D67D5EBED'" ng-bind="document.type.customValue|lstring:locale"></span>
 				</div>
 			</div>
-
+			<div ng-show="document.jurisdiction">
+				<label>Area of jurisdiction</label>
+				<div class="km-value">
+					<span ng-if="!document.jurisdiction.customValue && !document.jurisdictionScope" ng-bind="document.jurisdiction|term:locale"></span>
```
Comment (blaisefonseca) [inline app/views/forms/view/bch/view-biosafety-law.directive.html:41]:
```suggestion
					<span ng-if="!document.jurisdiction.customValue" ng-bind="document.jurisdiction|term:locale"></span>
```

**Review by blaisefonseca — COMMENTED** [review]
accept the change and rebase with master

File: app/views/forms/edit/bch/directives/edit-law.directive.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ define(['app', 'lodash', 'text!./edit-law.directive.html', 'views/forms/edit/edi
 	'views/forms/edit/document-selector', "views/forms/view/bch/view-biosafety-law.directive"], 
 function (app, _, template) {
 
-	app.directive("editBiosafetyLaw", ["$controller", "thesaurusService", "$q", "$filter", function($controller, thesaurusService, $q, $filter) {
+	app.directive("editBiosafetyLaw", ["$controller", "thesaurusService", "$q", "$filter","Enumerable", function($controller, thesaurusService, $q, $filter,Enumerable) {
```
Comment (sbilodeau) [inline app/views/forms/edit/bch/directives/edit-law.directive.js:5]: Are we still use enumerable?  Maybe we should get rid of this.

**Review by sbilodeau — APPROVED** [review]

File: app/views/forms/edit/bch/directives/edit-law.directive.js (hunk 2 above)
Comment (blaisefonseca) [reply app/views/forms/edit/bch/directives/edit-law.directive.js:5]: yes I am wondering myself, will remove in future

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**